spi/src/main/java/org/jboss/shrinkwrap/descriptor/spi/node/Node.java

```diff
@@ -222,6 +227,34 @@ public void setComment(final boolean comment) throws IllegalArgumentException {
     }
 
     /**
+     * Returns whether or not this {@link Node} represents a CDATA section
+     *
+     * @return
+     */
+    public boolean isCDATASection() {
+        return cdataSection;
+    }
+
+    /**
+     * Marks this {@link Node} as a CDATA section
+     *
+     * @param cdataSection
+     *            Whether or not this is a CDATA section
+     * @throws IllegalArgumentException
+     *             If this node has children
+     */
+    public void setCDATASection(final boolean cdataSection) throws IllegalArgumentException {
+        // Cannot have children
```

Comment:
Both comments in this method are not really needed for my taste as the code is self explanatory. Can you remove those?